engibench/problems/beams2d/v0.py

```diff
@@ -292,6 +292,7 @@ def reset(self, seed: int | None = None, **kwargs) -> None:
             **kwargs: Additional keyword arguments.
         """
         super().reset(seed, **kwargs)
+        self.__st = State()
```

Comment:
Isn't state reinitialized when calling optimize and simulate? e.g., see line 230.

Reply:
Sounds good